plugins/algoltek-aux/fu-algoltek-aux-firmware.c

```diff
@@ -32,7 +32,7 @@ fu_algoltek_aux_firmware_parse(FuFirmware *firmware,
 			       FwupdInstallFlags flags,
 			       GError **error)
 {
-	const gchar *version;
+	g_autofree gchar *version;
```

Comment:
these all need to be initialized with `= NULL` I'm afraid -- it's not needed to be correct, but static analysis gets very upset.

Reply:
Thanks! Right, I guess it would complain about potentially calling `g_free()` on an uninitialized value.